scripts/linters/pre_commit_linter.py

```diff
@@ -230,43 +219,47 @@ def _lint_all_files(
     css_file_extension_type = 'css' in file_extensions_to_lint
     other_file_extension_type = 'other' in file_extensions_to_lint
 
-    if not other_file_extension_type:
-        general_files_to_lint = '.' + file_extensions_to_lint
+    if js_ts_file_extension_type:
+        general_files_to_lint = _FILES['js'] + _FILES['ts']
     else:
-        general_files_to_lint = file_extensions_to_lint
+        general_files_to_lint = _FILES[file_extensions_to_lint]
 
     custom_linter, third_party_linter = general_purpose_linter.get_linters(
-        _FILES[general_files_to_lint],
+        general_files_to_lint,
         verbose_mode_enabled=verbose_mode_enabled)
     custom_linters.append(custom_linter)
 
     if js_ts_file_extension_type:
         custom_linter, third_party_linter = js_ts_linter.get_linters(
-            js_filepaths, ts_filepaths,
+            _FILES['js'], _FILES['ts'],
             verbose_mode_enabled=verbose_mode_enabled)
         custom_linters.append(custom_linter)
         third_party_linters.append(third_party_linter)
 
     if html_file_extension_type:
         custom_linter, third_party_linter = html_linter.get_linters(
-            html_filepaths, verbose_mode_enabled=verbose_mode_enabled)
+            _FILES['html'], verbose_mode_enabled=verbose_mode_enabled)
         custom_linters.append(custom_linter)
         third_party_linters.append(third_party_linter)
 
+        config_path_for_css_in_html = os.path.join(
+            parent_dir, 'oppia', '.stylelintrc')
         custom_linter, third_party_linter = css_linter.get_linters(
-            config_path_for_css_in_html, html_filepaths,
+            config_path_for_css_in_html, _FILES['html'],
```

Comment:
Do we pass the linter config for js/ts/py linters? (If not, those linters directly reads the config file? If yes then any specific reason of having a different flow for the CSS linters?)

Reply:
js/ts/py linters have their config file passed in their respective linter script. But we are passing CSS config files here because we have two config files one for CSS in HTML and one for general CSS linting.